src/ramanchada2/protocols/calibration.py

```diff
@@ -31,241 +34,245 @@ def set_model(self, model, model_units, peaks, name=None):
+        # else:
+        #     find_kw = dict(sharpening=None)
+        #     spe_pos_dict = spe_to_process.find_peak_multipeak(**find_kw).get_pos_ampl_dict()  # type: ignore
+        # prominence = prominence, wlen=wlen, width=width
         find_kw = dict(sharpening=None)
         if should_fit:
-            spe_pos_dict = spe_to_process.fit_peak_positions(center_err_threshold=10, 
-                                find_peaks_kw=find_kw,  fit_peaks_kw=fit_peaks_kw)  # type: ignore           
-            #fit_res = spe_to_process.fit_peak_multimodel(candidates=cand,**fit_peaks_kw)
-            #pos, amp = fit_res.center_amplitude(threshold=1)
-            #spe_pos_dict = dict(zip(pos, amp))        
+            spe_pos_dict = spe_to_process.fit_peak_positions(
+                    center_err_threshold=10, find_peaks_kw=find_kw, fit_peaks_kw=fit_peaks_kw)  # type: ignore
+            # fit_res = spe_to_process.fit_peak_multimodel(candidates=cand, **fit_peaks_kw)
+            # pos, amp = fit_res.center_amplitude(threshold=1)
+            # spe_pos_dict = dict(zip(pos, amp))
         else:
-            #prominence=prominence, wlen=wlen, width=width
+            # prominence = prominence, wlen=wlen, width=width
             cand = spe_to_process.find_peak_multipeak(**find_kw)
-            spe_pos_dict = cand.get_pos_ampl_dict()        
+            spe_pos_dict = cand.get_pos_ampl_dict()
+
+        ax[2].stem(spe_pos_dict.keys(), spe_pos_dict.values(), linefmt='b-', basefmt=' ')
+        ax[2].twinx().stem(self.ref.keys(), self.ref.values(), linefmt='r-', basefmt=' ')
 
-        ax[2].stem(spe_pos_dict.keys(),spe_pos_dict.values(),linefmt='b-', basefmt=' ')
-        ax[2].twinx().stem(self.ref.keys(),self.ref.values(),linefmt='r-', basefmt=' ')
-    
-        x_spe,x_reference,x_distance,df = rc2utils.match_peaks(spe_pos_dict, self.ref)
+        x_spe, x_reference, x_distance, df = rc2utils.match_peaks(spe_pos_dict, self.ref)
         sum_of_differences = np.sum(np.abs(x_spe - x_reference)) / len(x_spe)
         print("sum_of_differences original {} {}".format(sum_of_differences, self.ref_units))
-        if len(x_reference)==1:
-            _offset = ( x_reference[0] - x_spe[0])
-            print("ref",x_reference[0],"sample", x_spe[0],"offset", _offset, self.ref_units)
-            self.set_model(_offset, self.ref_units, df,name)
+        if len(x_reference) == 1:
+            _offset = (x_reference[0] - x_spe[0])
+            print("ref", x_reference[0], "sample", x_spe[0], "offset", _offset, self.ref_units)
+            self.set_model(_offset, self.ref_units, df, name)
         else:
-            fig, ax = plt.subplots(1,1,figsize=(3,3))
-            ax.scatter(x_spe,x_reference,marker='o')
+            fig, ax = plt.subplots(1, 1, figsize=(3, 3))
+            ax.scatter(x_spe, x_reference, marker='o')
             ax.set_xlabel("spectrum x ".format(self.ref_units))
             ax.set_ylabel("reference x ".format(self.ref_units))
```

Comment:
@vedina `self.ref_units` is supposed to be inserted in the string, but there's no placeholder `{}` in the string itself. Should it be something like `spectrum x ({})` (i.e. the units in parentheses)?